sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client.hpp

```diff
@@ -341,7 +342,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * @param name The name of the key.
      * @param context A #Azure::Core::Context controlling the request lifetime.
      */
-    Azure::Response<std::vector<uint8_t>> BackupKey(
+    Azure::Response<Azure::Security::KeyVault::Keys::BackupKey> BackupKey(
```

Comment:
@heaths and @tg-msft I think `BackupKey` is probably the right name here, how come .NET decided to call it `BackupKeyResult`?

Trying to page-back and rationalize the model type naming table again:
https://azure.github.io/azure-sdk/general_design.html#model-types

![image](https://user-images.githubusercontent.com/6527137/125145574-91e10b00-e0d6-11eb-89d7-b9d128ac71b7.png)

Reply:
We don't - we return a `byte[]` array. Why are you returning something else that the other SDKs don't do?

* https://github.com/Azure/azure-sdk-for-net/blob/c216595c466f15518f2ad8b6733841d3c8009370/sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs#L883
* https://github.com/Azure/azure-sdk-for-java/blob/0747a4129d3ae48fe2a8f54070a79fb3faf16913/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java#L656
* https://github.com/Azure/azure-sdk-for-python/blob/e75c11b16477185b344cb8cd672253c88a587a98/sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py#L511
* https://github.com/Azure/azure-sdk-for-js/blob/9656690691ac5aa3fd17eb3e334d9bceba51efc3/sdk/keyvault/keyvault-keys/src/index.ts#L623

Reply:
What about https://github.com/Azure/azure-sdk-for-net/search?q=BackupKeyResult?

Reply:
From offline discussion, that is from the old track 1 SDK, so ignore it.